### PR TITLE
rebuild: Adds rebuild script

### PR DESCRIPTION
**Pull request by Foxboron**
Handy script for going over tedious todo lists.

Signed-off-by: Morten Linderud <morten@linderud.pw>

### CODE REVIEW

File: package/rebuild [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+		    --multilib     Rebuild [multilib] packages
+
+		Examples:
+		    Rebuilds all packages in [community] towards [community] ignoring "archlinux-contrib"
+		    $  ${PROGNAME} -i "archlinux-contrib" "https://archlinux.org/todo/some-todo-list/"
+
+		    Rebuilds all packages from [extra] towards [extra-testing]
+		    $  ${PROGNAME} --extra --testing "https://archlinux.org/todo/some-todo-list/"
+
+		    Rebuilds all packages from [core] towards [extra-testing]
```
Comment (Foxboron) [inline package/rebuild:49]: Typo :c

File: package/rebuild-todo [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        ;;
+        esac
+        shift
+done
+
+TMPDIR=$(mktemp -d /var/tmp/contrib-rebuild.XXXX) || exit 1
+trap "rm -rf ${TMPDIR}" EXIT
+
+REBUILDLIST="$TMPDIR/rebuild_list"
+touch "${REBUILDLIST}"
```
Comment (Foxboron) [inline package/rebuild-todo:170]: Needs to be removed

**Comment by anthraxx** [comment]
Wouldn't be a better place for a generic rebuild script within devtools? There is also `rebuildpkgs`. Having a good rebuild script in devtools would IMO make more sense then fragmenting it into the contrib package.

**Comment by Foxboron** [comment]
I don't see any rush promoting it to devtools. Having it in contrib with some documentation and playing around with it then adding it to devtools seems like a fine solution as well.

**Pull request merged**